### PR TITLE
clean images and set version in bin/docker-up-vinyldns.sh

### PR DESCRIPTION
**Pull request by nimaeskandary**
Changes in this pull request:
- allows a user to purge vinyldns docker images used by `bin/docker-up-vinyldns.sh`
- allows a user to overwrite the version tag used by `bin/docker-up-vinyldns.sh`

### CODE REVIEW

File: bin/docker-up-vinyldns.sh [hunk 1]
```diff
@@ -49,36 +45,67 @@ function usage {
...
-        -t | --timeout ) TIMEOUT="$2";  shift;;
-        -a | --api-only ) SERVICE="api";;
-        * ) usage; exit;;
-    esac
-    shift
-done
+function clean_images {
+	if (( $CLEAN == 1 )); then
+		echo "cleaning docker images..."
+		docker rmi vinyldns/api:"$VINYLDNS_VERSION"
```
Comment (mitruly) [inline bin/docker-up-vinyldns.sh]: Would it make more sense to remove all `vinyldns/api` and `vinyldns/portal` images? Could pass in `$(docker images -a | grep vinyldns/api | awk '{print $3}')` (image ID list) to `docker rmi` if that's the case.

File: docker/.env [hunk 2]
```diff
@@ -14,3 +14,6 @@ DYNAMODB_ENDPOINT=http://vinyldns-dynamodb:8000
 USER_TABLE_NAME=users
 USER_CHANGE_TABLE_NAME=userChange
 TEST_LOGIN=true
+
+# api and portal docker tag
+VINYLDNS_VERSION=0.8.0
```
Comment (rebstar6) [inline docker/.env]: should this instead work on the `latest` tag?

File: bin/docker-up-vinyldns.sh (hunk 1 above)
Comment (nimaeskandary) [reply bin/docker-up-vinyldns.sh]: hm I would prefer deleting only the images used since others installed wouldn't be involved in the run

Comment (mitruly) [reply bin/docker-up-vinyldns.sh]: Yeah, I was thinking about that as well. Realistically, people would want to be using a single image (either snapshot or release) and wouldn't have multiple.

With that said, I'm fine with having `clean` remove only the version that matches the script execution if we update the help text to something like:
```
-c, --clean: re-pull image versions from docker hub
```

**Pull request merged**